ledger-core/runner/runner.go

```diff
@@ -173,15 +182,15 @@ func (r *DefaultService) executeMethod(
 
 	classReference, err := objectDescriptor.Class()
 	if err != nil {
-		return nil, errors.W(err, "couldn't get class reference")
+		return nil, errors.W(err, "couldn't get class reference", ErrorDetail{DetailEmptyClassRef})
```

Comment:
could you switch to direct throw usage?

Reply:
yes, errors -> throw
this "errors" name is a remnant of previous rewrite to remove /x/errors

Reply:
Replaced `errors` package with `throw` everywhere in `runner.go`